src/Unosquare.Labs.EmbedIO/Core/ReverseOrdinalStringComparer.cs

```diff
@@ -6,14 +6,14 @@
     // Sorts strings in reverse order to obtain the evaluation order of virtual paths
     internal sealed class ReverseOrdinalStringComparer : IComparer<string>
     {
-        private static readonly IComparer<string> _directComparer = StringComparer.Ordinal;
+        private static readonly IComparer<string> DirectComparer = StringComparer.Ordinal;
```

Comment:
I don't know if you could use **StringComparer.OrdinalIgnoreCase** instead of **StringComparer.Ordinal**